dependencies.qmd

```diff
@@ -34,6 +34,21 @@ We carefully evaluate the potential benefits of data.table as a dependency based
 
 We embrace an [iterative development approach](principles.html#lean-and-agile-collaboration-framework) where dependencies can evolve over time. Initially, dependencies can be included to quickly build a Minimum Viable Product (MVP) and gain early feedback. However, we recognize that not all dependencies may bring significant value in the long-run. Therefore, we encourage package maintainers to regularly evaluate the usefulness and impact of dependencies and remove those that do not contribute substantially to the package. The [itdepends package](https://github.com/r-lib/itdepends) can help to identify dependencies that are used just one time in a given package or codebase. Such dependencies are good candidates for removal.
 
+### Base R support schedule
+
+We maintain a minimum support for the last four versions of R ([the {tidyverse} standard](https://www.tidyverse.org/blog/2019/04/r-version-support/)). The table outlines the concrete minimum supported version.
```

Comment:
```suggestion
We maintain a minimum support for the last four versions of R ([the {tidyverse} standard](https://www.tidyverse.org/blog/2019/04/r-version-support/)). The table outlines the concrete minimum supported version.
Adopting the tidyverse standard allows us to use relatively recent R features, while putting a low pressure on our users to update their R version since they will already have to update to use the latest ggplot2 or tidyverse version.
```

Feel free to rephrase but it'd be great to have some explanation of the rationale.